templates/docker-ingress.json

```diff
@@ -0,0 +1,43 @@
+{
+  "apiVersion": "extensions/v1beta1",
+  "kind": "Ingress",
+  "metadata": {
+    "annotations": {
+      "kubernetes.io/ingress.class": "nginx",
+      "nginx.ingress.kubernetes.io/ssl-redirect": "false",
+      "nginx.ingress.kubernetes.io/proxy-body-size": "0",
```

Comment:
I guess YAML would allow to add an inline comment here to explain why we need to set `proxy-body-size` to `0`.

Reply:
changes applied